spec/statesman/adapters/active_record_spec.rb

```diff
@@ -163,6 +163,39 @@
             to(change { previous_transition.reload.updated_at })
         end
 
+        context "with a custom updated timestamp column set" do
+          before do
+            RemoveUpdatedAtFromMyActiveRecordModelTransitions.migrate(:up)
+            AddUpdatedOnToMyActiveRecordModelTransitions.migrate(:up)
+            MyActiveRecordModelTransition.updated_timestamp_column = :updated_on
+            MyActiveRecordModelTransition.reset_column_information
+          end
+
+          after do
+            MyActiveRecordModelTransition.updated_timestamp_column =
+              MyActiveRecordModelTransition::DEFAULT_UPDATED_TIMESTAMP_COLUMN
+          end
+
+          it "touches the previous transition's updated_on timestamp" do
+            expect { Timecop.freeze(Time.now + 5.seconds) { create } }.
+              to(change { previous_transition.reload.updated_on })
+          end
+        end
+
+        context "with no updated timestamp column set" do
+          before { MyActiveRecordModelTransition.updated_timestamp_column = nil }
+
+          after do
+            MyActiveRecordModelTransition.updated_timestamp_column =
+              MyActiveRecordModelTransition::DEFAULT_UPDATED_TIMESTAMP_COLUMN
+          end
+
+          it "just updates the most_recent" do
+            expect { Timecop.freeze(Time.now + 5.seconds) { create } }.
+              to(change { previous_transition.reload.most_recent })
+          end
```

Comment:
Should we test here that we're not updating `updated_at`?

Reply:
Yes - great shout!